docs/howto/openstack/neutron/create-security-groups.md

```diff
@@ -0,0 +1,55 @@
+# Creating security groups
+
+[By definition](https://docs.openstack.org/nova/latest/admin/security-groups.html), security groups are "[...] _sets of IP filter rules that are applied to all project instances, which define networking access to the instance. Group rules are project specific; project members can edit the default rules for their group and add new rule sets_".
+
+To create a security group click on _Security Groups_ in the left-hand navigation menu:
+
+![navigation-panel](assets/create-security-groups/01-navigation-panel.png)
+
+and then click on _Create new Security Group_ in the top-right corner:
+
+![create-button](assets/create-security-groups/02-create-button.png)
+
+> An alternative way to create a Security Group is by clicking on _Create ...._ button in the top bar.
+
+Now give the security group a name and description, and choose in which region to create it, then click _create_:
+
+![create-panel](assets/create-security-groups/03-name-creation.png)
+
+## Remove default ingress rules
+
+By default, a security group named `default` has been already created for you, allowing all traffic from any source (ingress), and to any destination (egress).  
+Click on it and select the _Rules_ tab to view its rules:
+
+![default-rules](assets/create-security-groups/04-default-rules.png)
+
+We recommended to either create and use a new security group than the default one, **or** restrict ingress traffic to specific ports and sources.
+
+If you want use the default group, **remove the two ingress rules** that allow all incoming traffic.
+
+![default-ingress-rules](assets/create-security-groups/05-default-ingress-rules.png)
+
+## Allow SSH access
+
+The next thing to do, is to allow SSH access on **port 22** to the server, only from specific VPN(s). To do this click on the _Create new rule_ button.
```

Comment:
I would use "only from specific networks" here. (Needn't be a VPN).

Also, it should be "To do this, click" (with a comma).